From 37d9ee45971fc41828406dfac176c2b7a6a242c0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E2=80=9Cdoti=E2=80=9D?= <“doti@catboyindustries.co”>
Date: Tue, 12 May 2026 21:33:07 -0700
Subject: [PATCH] use raspberry pi linux source

---
 Pkgfile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Pkgfile b/Pkgfile
index 7c60d3d..f651612 100644
--- a/Pkgfile
+++ b/Pkgfile
@@ -97,10 +97,10 @@ vars:
   kspp_sha256: 640b25473f86557d082f680b0f57653a69f4146e97825673bf582f9a4c8b7739
   kspp_sha512: 82dae1debbe94a3f82766a8cdbfe59ff8698d433175803458499b76c50b47b89e9280b677a9f16b2a44711badd6f4001aba951a41722a74dd51c03a18b8b9219
 
-  # renovate: datasource=git-tags extractVersion=^v(?<version>.*)$ depName=git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git
-  linux_version: 6.18.24
-  linux_sha256: c207c557ce58103b4dda30e26da5203f3d8467c6dadc53d709f6d83ae1d1255f
-  linux_sha512: ad734fae886272f0c22853c885f618711baa2128b6447d6b996c884a16781893b73c65f5cf58087637a44d33ced1e5ecff9ead2fbca0bbeb7686dbbd4628de98
+  # rpi-6.18.y branch, commit 5fe081e028463bc519d86ff712224d1f489a9e25 (2026-05-12), kernel 6.18.29
+  raspberrypi_kernel_version: 5fe081e028463bc519d86ff712224d1f489a9e25
+  raspberrypi_kernel_sha256: a0a19336348f864f21ee066a49a92ddcab5f9a096f2a023aae442cd5e445cc05
+  raspberrypi_kernel_sha512: 74a65bb11f09a23c69f153961399760a4b2237f9e9f78f4ba27379743a583387e0665e67ddb4c465922356b5213401be649a32b9bc2bb3ae59eab2d8620be23d
 
   # renovate: datasource=git-tags extractVersion=^libaio-(?<version>.*)$ depName=https://pagure.io/libaio.git
   libaio_version: 0.3.113
-- 
2.54.0

